config/lib/jwtAuthentication.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var config = require('../config'),
+  jwt = require('jsonwebtoken');
+
+// build token auth service
+var tokenAuth = {
+  signToken: signAuthToken
+};
+
+// export the token auth service
+module.exports = tokenAuth;
```

Comment:
- in the names instead of camelcase. I don't see any good reason for it to be in either. @codydaig any suggestions on file location?